src/cinder/Log.cpp

```diff
@@ -318,6 +409,44 @@ void LoggerNSLog::write( const Metadata &meta, const string &text )
 	NSLog( @"%@ %@", metaDataNs, textNs );
 }
 
+// ----------------------------------------------------------------------------------------------------
+// MARK: - LoggerNSLog
+// ----------------------------------------------------------------------------------------------------
+
+LoggerSysLog::LoggerSysLog()
+{
+	// Most general approach for determining app name.  Perhaps there is something more cinder specific?
+	// https://developer.apple.com/library/mac/qa/qa1544/_index.html
```

Comment:
I don't think we store an app name string anywhere, perhaps we should. Though if we keep the approach here, we should switch to using [PlatformCocoa::getBundle()](https://github.com/cinder/Cinder/blob/app_refactor/include/cinder/app/cocoa/PlatformCocoa.h#L60) once app_refactor is merged.